.gitignore

```diff
@@ -173,6 +173,8 @@ uptime-*.json
 /src/lib/libtor-ctime-testing.a
 /src/lib/libtor-err.a
 /src/lib/libtor-err-testing.a
+/src/lib/libtor-intmath.a
```

Comment:
What about a `math` module instead and we put everything related to "math" in there (not only int stuff). I mean with this name, does the floating math functions apply to go in?

Reply:
no, higher-level stuff should go in another module.

This one is for low-level integer math.